static-instrumenter/agent-instrumenter/src/integrationTest/java/JarTest.java

```diff
@@ -43,17 +46,25 @@ void testSampleJar() throws Exception {
             outPath.toString());
 
     Process instrumentationProcess = instrumentationProcessBuilder.start();
-    InputStream instrumentationIn = instrumentationProcess.getErrorStream();
     instrumentationProcess.waitFor(10, TimeUnit.SECONDS);
 
-    String instrumentationLog =
-        new String(instrumentationIn.readAllBytes(), StandardCharsets.UTF_8);
+    boolean isHotJdkJvm = System.getProperty("java.vm.name").contains("OpenJDK");
+    if (isHotJdkJvm) {
```

Comment:
Does the instrumentation process print out output only on OpenJDK? I'm wondering what's the reason behind this condition

Reply:
It's to check that the java agent is started. This message is generated for OpenJDK. I would prefer to have this check to fail fast if something unexpected happens.

Reply:
could we check for `INFO io.opentelemetry.javaagent.tooling.VersionLogger - opentelemetry-javaagent` instead? I think the purpose of that check would probably be clearer than checking the class sharing message

Reply:
Done. I was not aware of this OTel message. It's better. However, this message and the debug messages are printed into the error output. Do you know why?